### ISSUE TITLE
Loss is decreasing and predicting data but Accuracy not increasing.

### ISSUE THREAD

**Issue by Mohanrajs** [keras-team/keras#10329]
Loss in LSTM network is decreasing and predicting time series data closer to existing data but Accuracy is increased to some value like acc - 0.784 and constantly repeating for all the Epochs 
or else
There is another possibility will be like accuracy will be 0 for all the epochs neither it's increasing nor it's decreasing.

**Comment by kvignesh1420** [comment]
Try increasing the batch size and reduce the learning rate while training. Please let me know.

**Comment by Mohanrajs** [comment]
Yup, done it. But now entirely different as it changed accuracy into fluctuating form (increasing and decreasing) for "N" set of epochs performing the training. 

Though this is weird as the existing data (Historical data) and predicted data is closer to each other in the graph and loss is decreasing in the console.

One more detail is I am using "ModelCheckpoint"  from keras to save the best model and reload it if training restarts.

**Comment by kvignesh1420** [comment]
Can you post the screen-shot of the training process.

**Comment by Mohanrajs** [comment]
Yup,
###  Images are in order wide placed.
I figured out using dropout increases the fluctuations.
![1_training epoch set_1](https://user-images.githubusercontent.com/2352257/40900011-49510bb2-67e7-11e8-9a84-e6d3ae046e60.png)
![2_training epoch set_2](https://user-images.githubusercontent.com/2352257/40900047-5d3fcd8e-67e7-11e8-96c0-d4478392e55e.png)
![3_training epoch epoch increased from 20 to 75 set_3](https://user-images.githubusercontent.com/2352257/40900058-6ad953b6-67e7-11e8-86ad-91efbaf0a4a8.png)
![4_training epoch epoch increased from 20 to 75 set_4___with increased in graph accuracy](https://user-images.githubusercontent.com/2352257/40900069-73f7beb0-67e7-11e8-85eb-eaa1db9c9465.png)
![5_training epoch epoch decreased from 75 to 20 set_12](https://user-images.githubusercontent.com/2352257/40900077-7d7f18fc-67e7-11e8-8c15-73ad7398f6ae.png)
![6_layers used](https://user-images.githubusercontent.com/2352257/40900091-8abcd05e-67e7-11e8-80e0-58e38d0b53c4.png)

**Comment by kvignesh1420** [comment]
You don't need to consider accuracy as a metric, as this is a regression problem. A decrease in loss function is what you need to consider in the best model.  Also, use a lower learning rate to start with (just a suggestion)

**Comment by Mohanrajs** [comment]
Okay, I'll try that and thank you for your suggestion.

**Comment by navodikamadushan** [comment]
same issue for me

**Comment by navodikamadushan** [comment]
I faced the same issue
